CMake/Collage.spec

```diff
@@ -1,13 +1,13 @@
 Name:		Collage
-Version:	1.2.1
+Version:	
 Release:	1%{?dist}
 Summary:	Cross-platform C++ network library
 
 Group:		Development/Libraries
 License:	LGPLv2
 URL:		http://www.libcollage.net/
 Source0:	http://www.equalizergraphics.com/downloads/%{name}-%{version}.tar.gz
-Patch0:		Collage-1.2.1-build-fix.patch
+Patch0:		Collage--build-fix.patch
```

Comment:
Please revert this file

Reply:
Ok - done. Not sure how the version got lost in the first place, but I added it back.